### PR TITLE
Redesign Gnana Page

### PR DESCRIPTION
**Pull request by babadrape**
- [x] add Gnana Utility component
- [x] add Gnana Disclaimers Component
- [x] make Utility and Disclaimers component mobile responsive
- [x] add UIKit Icon to the Gnana Utility
- [x] redesign the Gnana Page
- [x] make checkbox square
- [x] move warning text to center in both mobile and desktop screens
- [x] move styles to own files

**REVIEW CHANGES REQUESTS**
- [x] change Header Title to Poppins and centralize
- [x] make unselected checkbox background grey in dark mode
- [x] change font weight on ReturnCard and ConverCard to match design
- [x] resolve font-size and font-weight issues with GNANA Utility and Disclaimers sections to fit design as best as possible
- [x] change the text of the disclaimers section
- [x] remove IAO and POOLS component if it isn't being used
- [ ] make sure isApproved state is working on convert and return cards

### CODE REVIEW

File: src/views/ApeZone/components/Pools/Pools.tsx [hunk 1]
```diff
@@ -1,275 +1,35 @@
...
+  LabelWrapper,
+  StyledText,
+  ButtonCheckWrapper,
+  ToggleContainer,
+  ToggleWrapper,
+  StyledCheckbox,
+} from './styles'
 
 const Pools: React.FC = () => {
   const [stakedOnly, setStakedOnly] = useState(false)
```
Comment (Harambe-Nakamoto) [inline src/views/ApeZone/components/Pools/Pools.tsx:35]: Is this file being used anymore?

**Review by Harambe-Nakamoto — COMMENTED** [review]
Nice work! Just had one question

File: src/views/ApeZone/components/Pools/Pools.tsx (hunk 1 above)
Comment (babadrape) [reply src/views/ApeZone/components/Pools/Pools.tsx:35]: I remember we used to have a section in the nav with title **GNANA POOLS** but I don't see it anymore. I think this is what this component is for but if that page isn't in existence anymore then this isn't needed. However, I'll check on it and be certain about it.

Comment (Harambe-Nakamoto) [reply src/views/ApeZone/components/Pools/Pools.tsx:35]: There is also an IAO component that I don't think is being used anymore. If so could you remove both of them?

Comment (babadrape) [reply src/views/ApeZone/components/Pools/Pools.tsx:35]: Also, DK made some change requests to be done, that's what I'm working on now before it's ready for final review

**Comment by babadrape** [comment]
> Nice work! Just had one question

Is this the question you asked above?

File: src/views/ApeZone/components/Pools/Pools.tsx (hunk 1 above)
Comment (babadrape) [reply src/views/ApeZone/components/Pools/Pools.tsx:35]: > There is also an IAO component that I don't think is being used anymore. If so could you remove both of them?

Is this the same IAO component that is being moved to the IAO page that will be released on the 20th?

**Pull request merged**